### ISSUE TITLE
64bit linux support

### ISSUE THREAD

**Issue by Thandriel** [erfg12/clanmod-jka#5]
While running on 64bit linux machine (using openJK) it throws:

```
Sys_LoadGameDll(/home/than/.local/share/openjk/clanmod/jampgamex86_64.so) failed: "/home/than/.local/share/openjk/clanmod/jampgamex86_64.so: cannot open shared object file: No such file or directory"
Sys_LoadGameDll(./clanmod/jampgamex86_64.so) failed: "./clanmod/jampgamex86_64.so: cannot open shared object file: No such file or directory"
Sys_LoadGameDll(/home/than/.local/share/openjk/base/jampgamex86_64.so) failed: "/home/than/.local/share/openjk/base/jampgamex86_64.so: cannot open shared object file: No such file or directory"
VM_Create: jampgamex86_64.so succeeded
```
The mod provides only jampgamei386.so.
Most things seems to work nevertheless, but every clanmod command throws unknown command.

**Comment by erfg12** [comment]
Do you have GCC? Try compiling the code to x86_64. Currently I'm at work
and I don't have a Linux OS handy.

You can try this, I pulled these notes from some compilation guide on the
internet...

Linux/Mac OS X CMake notes

Building on Linux or Mac OS X requires SDL2.
If you wish to build on 64-bit Linux, you should force it to build 32-bit
for now, by adding the CMake flags -DCMAKE_CXX_FLAGS=-m32
-DCMAKE_C_FLAGS=-m32 -DCMAKE_SHARED_LINKER_FLAGS=-m32
-DCMAKE_SIZEOF_VOID_P=4

Basically changing the makefile to write 32/64 bit code instead of i386.
No idea if it will work or not.

**Comment by Thandriel** [comment]
It seems it was sort of pebkac problem :) Tried compiling myself even before I created this ticked and it didn't solve it. Tried again today and it works. Of course I had to rename the file from i386 to x86_64. FYI I succeeded compiling the library just with make command, no arguments needed and with provided gcc makefile. I'm on ubuntu 15.04 x86_64.

**Comment by erfg12** [comment]
If you get a chance, give me a copy and I'll distribute it with the next build package.

**Comment by Thandriel** [comment]
[Here](https://drive.google.com/open?id=0B1zbWhHUHE68R2d2OGRhRElGSHc) you go. Sorry for delay :)

**Comment by erfg12** [comment]
Ok, I requested access. Need approval.

**Comment by Thandriel** [comment]
Weird. Anyway access granted.
